.github/workflows/full-site-editing-plugin.yml

```diff
@@ -28,6 +28,11 @@ jobs:
       if: steps.cache.outputs.cache-hit != 'true'
       run: yarn install --frozen-lockfile # Not needed when restoring from cache.
 
+    - name: Run composer
+      uses: nick-zh/composer-php@master
+      with:
+        action: 'install'
```

Comment:
Are Composer packages cached in Circle CI?

Reply:
Could you elaborate on that question? We're using Github Actions here, no?

Reply:
🤦 hah, of course no circle ci here.

So e.g. `node_modules` gets cached so that running `yarn` is faster, and I was wondering if results from `composer install` also could get cached?

Reply:
Ah I am by no means an expert in this realm. All I can say that the time added by the composer install was really minimal, so I am not sure if it's worth trying to cache something.

Reply:
![Screenshot 2020-07-02 at 11 22 16](https://user-images.githubusercontent.com/203408/86341249-522b4c00-bc56-11ea-9f8c-eebe3605836a.png)

Reply:
Sounds good then! :-) 👍 Thanks for checking the time.